src/Network/Response.php

```diff
@@ -538,13 +538,18 @@ protected function _sendHeader($name, $value = null)
 
     /**
      * Sends a content string to the client.
+     * If the content is a callable, it is invoked.
      *
      * @param string $content string to send as response body
      * @return void
      */
     protected function _sendContent($content)
     {
-        echo $content;
+        if (!is_string($content) && is_callable($content)) {
```

Comment:
The "not a string" check will prevent using a static method as callback provided as `SomeClass::someStaticMethod`.